.travis.yml

```diff
@@ -1,3 +1,5 @@
 language: objective-c
 osx_image: xcode9.3
+before_install:
+- pod repo update
```

Comment:
Hehe yep, now that the pods are not part of the repo anymore we have to update them manually, of course.